packages/clay-layout/src/index.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import Col from './Col';
+import Container from './Container';
+import ContainerFluid from './ContainerFluid';
+import {ContentCol, ContentRow, ContentSection} from './Content';
+import Row from './Row';
+import Sheet from './Sheet';
+
+export {
+	Col,
+	Container,
+	ContainerFluid,
+	ContentCol,
+	ContentRow,
+	ContentSection,
+	Row,
+	Sheet,
+};
+
+export default {
+	Col,
+	Container,
+	ContainerFluid,
+	ContentCol,
+	ContentRow,
+	ContentSection,
+	Row,
+	Sheet,
```

Comment:
What's the reason behind grouping all `Sheet` components but not all `Content` ones?

Reply:
Thats my bad, I will update it. The thought behind it is that we don't want multiple namespaces, so instead of `<ClayLayout.Sheet.Header>` it should be `<ClayLayout.SheetHeader`. So generally, the only namespace should be the top level component, in this case `ClayLayout`

Reply:
Just updated